gen3-client/commonUtils/commonUtils.go

```diff
@@ -105,6 +105,7 @@ type RetryObject struct {
 	GUID         string
 	RetryCount   int
 	Multipart    bool
+	Bucket *string
```

Comment:
Go conventions prefer the type to be all lined up

Reply:
Also, why not just use a string instead of a string pointer?

Reply:
using just string was giving me some weird errors when I tried to build it. Im not too familiar with it but I can change it to a string and give it another Go. 🥁